update.py

```diff
@@ -1 +1 @@
-print*"hello")
+print("hello")
```

Comment:
Reviewing changes in **update.py**:

Corrected syntax for print function, good job!